.github/workflows/validate-manifest.yaml

```diff
@@ -0,0 +1,24 @@
+name: Validate Manifest Schema
+
+on:
+  pull_request:
+    paths:
+      - 'compute/manifest.yaml'
+      - 'compute/manifest.schema.json'
```

Comment:
FYI: this workflow won't be blocking in the current setup.
The job/workflow might fail, but it will still be possible to merge it (on purpose or accidentally, with auto-merge enabled, for example).

I would add it to `.github/workflows/build_and_test.yml` if we want to make it impossible to merge the PR with the failed job.